pkg/exposer/csi_snapshot.go

```diff
@@ -259,7 +259,13 @@ func (e *csiSnapshotExposer) Expose(ctx context.Context, ownerObject corev1api.O
 		return errors.Wrap(err, "error to create backup pod")
 	}
 
-	curLog.WithField("pod name", backupPod.Name).WithField("affinity", csiExposeParam.Affinity).Info("Backup pod is created")
+	curLog = curLog.WithField("pod name", backupPod.Name)
```

Comment:
The `pod name` and `affinity` should only included by the current log `Backup pod is created`.

Even though at present no one is further using `curLog`, considering future development, the two fields should not be added into `curLog` permanently.

Reply:
Good catch